.github/workflows/ci-e2e.yml

```diff
@@ -0,0 +1,76 @@
+name: E2E testing
+
+on:
+  push:
+    branches: [main, "release/*"]
+  pull_request:
+    branches: [main, "release/*"]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  pytester:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
How much does the workflow differ? Could it be just one additional parameter/switch?